### PR TITLE
Synchronous standbys

### PR DESCRIPTION
**Pull request by zilder**
## What
A wrapper for [synchronous_standby_names](https://www.postgresql.org/docs/current/static/runtime-config-replication.html#GUC-SYNCHRONOUS-STANDBY-NAMES) server option.
## Usage
```python
from testgres import get_new_node, First

master = get_new_node().init().start()
with master.replicate().start() as standby:
    master.append_conf("synchronous_commit = remote_apply")
    master.set_synchronous_standbys(First(1, [standby]))
    master.restart()
```

### CODE REVIEW

File: testgres/__init__.py [hunk 1]
```diff
@@ -24,3 +24,7 @@
     get_bin_path, \
     get_pg_config, \
     get_pg_version
+
+from .standby import \
+    First, \
```
Comment (funbringer) [inline testgres/__init__.py]: IMHO those names are too generic to be exported at top level.

**Comment by funbringer** [comment]
I think we should replace `First` and `Any` with a `Enum`-based class.

**Comment by funbringer** [comment]
Now we could write

```python
node.append_conf(synchronous_standby_names=First(1, [standby]))
```

File: testgres/__init__.py (hunk 1 above)
Comment (ildus) [reply testgres/__init__.py]: I think names are ok but should be used with the of module, like `standby.First`

Comment (funbringer) [reply testgres/__init__.py]: @ildus I agree.

**Pull request merged**